Users/Docker_Image.rst

```diff
@@ -33,6 +72,9 @@ on your code with a ``.gitlab-ci.yml``, like this:
   For more information about GitLab CI configuration, consult the
   `official documentation <https://docs.gitlab.com/ce/ci/>`__.
 
+Downloading Docker images and pushing images over low bandwidth connections
+is very slow.
```

Comment:
Okay my bad, it doesn't fit here nicely as well :3

However, I would actually rephrase it, to something like:
```
Docker images are usually very large. Downloading or pushing them over low bandwidth connections can be very slow.
```
What do you say? :)

And maybe we put it inside a `note::` back into ""What is a docker image  and how is it different from a container?"

Reply:
@Makman2 yes I think we should put it back in What is a docker image section